packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/camera.dart

```diff
@@ -331,3 +331,13 @@ class CameraUpdateZoomTo extends CameraUpdate {
   @override
   Object toJson() => <Object>['zoomTo', zoom];
 }
+
+/// Defines a animation configuration for camera updates.
+@immutable
+class CameraUpdateAnimationConfiguration {
+  /// Creates a immutable animation configuration for camera updates.
+  const CameraUpdateAnimationConfiguration({this.duration});
+
+  /// The duration of the animation.
```

Comment:
Let's document what a null value means. (I assume it's whatever the Google Maps SDK decides is a good default?)

Reply:
Updated the documentation to clarify the null case, does this work for you?